collectors/30/api_http_v2.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+import time
+import requests
+import sys
+import re
+
+from collectors.lib.optimizely_utils import format_tsd_key, get_json
+
+# Constants
+TIME = int(time.time())
+URL = 'http://127.0.0.1:8081/metrics'
+METRIC_PREFIX = 'optimizely.codahale.'
+
+def main():
+    json = get_json(URL)
+    res_mode_pattern = re.compile(r"results-(\w+)")
+    res_service_pattern = re.compile(r"com.optimizely(?:\.[a-z]+)+\.((?:[A-Z][a-z]+)*ResultsService)")
```

Comment:
I don't think we need to special case results service like this since the various ResultsService implementations have distinct metrics.